recipes/owlrl/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "owlrl" %}
+{% set version = "5.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - folder: dist
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 42c454b42027fa6955b74db11b5f0fee29281a1a99eadec5687bc66effe5c446
+    patches:
+      - 0001-remove-scripts.patch
+  - folder: src
+    url: https://github.com/RDFLib/OWL-RL/archive/v{{ version }}.tar.gz
+    sha256: ad7b8a4d87be6f31a5d20bc53ed418fc5bc7ac752617bffc85607f7793ff11cc
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - owlrl = owlrl._cli:main
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - rdflib >=4.2.2
+    - rdflib-jsonld >=0.4.0
+
+test:
+  source_files:
+    - src/tests
+  requires:
+    - pytest-cov
+  imports:
+    - owlrl
+  commands:
+    - owlrl --help
+    - cd src && cd tests && pytest --cov=owlrl -k "not test_dt_not_type" -vv
+
+about:
+  home: https://github.com/RDFLib/OWL-RL
+  license: W3C
+  license_family: OTHER
+  license_file: {{ environ["RECIPE_DIR"] }}/w3c.txt
```

It will automatically fall back to looking there:
```suggestion
  license_file: w3c.txt
```

---

So convenient!